pkg/api/vlabs/validate.go

```diff
@@ -276,6 +276,12 @@ func (a *Properties) ValidateOrchestratorProfile(isUpdate bool) error {
 					}
 				}
 
+				if o.KubernetesConfig.LoadBalancerSku != "" {
```

Comment:
We were never actually doing basic SKU value validation for vlabs. Here we check against case-insensitive variants of "basic" and "standard"